FindYourMovie/Controllers/MovieViewController.swift

```diff
@@ -0,0 +1,222 @@
+//  Copyright © 2020 Andres Guevara Caprio. All rights reserved.
+//
+
+import UIKit
+
+/// Controller for main view. It is a Table View that contains a search controller to search for movies through a movie databse API.
+/// The last 10 movies are persisted locally and displayed as a suggestion when the user searches for movies
+class MovieViewController: UITableViewController {
+    
+    var movieSearch = [Movie]()
+    var currentPage = 1
+    var currentSearch = ""
+    var totalPages = 1
+    var last10searches = [String]()
+    var lastSearchesController : MovieSearchController!
+    var searchController : UISearchController!
+    let defaults = UserDefaults.standard
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        // Create an instance of the second controller once main controller loads
+        lastSearchesController = MovieSearchController()
+        // Make the second controller a search result controller
+        searchController = UISearchController(searchResultsController: lastSearchesController)
+        searchController.searchResultsUpdater = self
+        if #available(iOS 13.0, *) {
+            searchController.searchBar.searchTextField.placeholder = "Search for a Movie"
+        } else {
+            searchController.searchBar.placeholder = "Search for a Movie"
+        }
+        // Add search controller to navigation controller
+        navigationItem.searchController = searchController
+        navigationItem.hidesSearchBarWhenScrolling = false
+        searchController.delegate = self
+        searchController.searchBar.delegate = self
+        definesPresentationContext = true
+        // Load latestes searches at loading time
+        if let searches = self.defaults.array(forKey: "lastSearches") as? [String] {
+            self.last10searches = searches
+        }
+    }
+    
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return movieSearch.count
+    }
+    
+    override func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "MovieCell", for: indexPath) as! MovieCell
+        // Configure each cell
+        let movie = movieSearch[indexPath.row]
+        cell.name.text = movie.title
+        cell.releaseDate.text = movie.release_date
+        cell.overview.text = movie.overview
+        cell.poster.load(string: movie.poster_path)
+        return cell
+    }
+    
+    override func tableView(_ tableView: UITableView, willDisplay cell: UITableViewCell, forRowAt indexPath: IndexPath) {
+        let currentPosition = indexPath.row
+        let totalRows = movieSearch.count
+        // If the user has scrolled more than 90% of the list, then get the next available page, if available
+        if (self.currentPage < self.totalPages && Double(currentPosition)/Double(totalRows) >= 0.9) {
```

Comment:
Usually in swift we don't put () around an if statement like this (but ok, that's mostly style). 
I realize there is a comment, but`0.9` is a magic number, can you create a descriptive named var for it at the class level?